roles/weaviate/templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,131 @@
+services:
+  traefik:
+    image: traefik:v3.2
+    container_name: traefik
+    restart: unless-stopped
+    security_opt:
+      - no-new-privileges:true
+    networks:
+      - weaviate_network
+    ports:
+      - "80:80"
+      - "443:443"
+    environment:
+      - LETSENCRYPT_EMAIL=${LETSENCRYPT_EMAIL}
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ./traefik/traefik.yml:/traefik.yml:ro
+      - ./traefik/acme.json:/acme.json:rw
+      - ./traefik/config.yml:/config.yml:ro
+    healthcheck:
+      test: ["CMD", "traefik", "healthcheck", "--ping"]
+      interval: 10s
+      timeout: 5s
+      retries: 3
+      start_period: 5s
+    deploy:
+      resources:
+        limits:
+          cpus: '{{ weaviate_traefik_cpu_limit }}'
+          memory: {{ weaviate_traefik_memory_limit }}
+        reservations:
+          cpus: '{{ weaviate_traefik_cpu_reservation }}'
+          memory: {{ weaviate_traefik_memory_reservation }}
+
+  weaviate:
+    image: cr.weaviate.io/semitechnologies/weaviate:${WEAVIATE_VERSION:-{{ weaviate_build_version }}}
+    container_name: weaviate
+    restart: unless-stopped
+    security_opt:
+      - no-new-privileges:true
+    networks:
+      - weaviate_network
+    ports:
+      - "50051:50051"
+    command:
+      - --host
+      - 0.0.0.0
+      - --port
+      - '8080'
+      - --scheme
+      - http
+    volumes:
+      - weaviate_data:/var/lib/weaviate
```

Comment:
Please use the path defined in weaviate_data_directory (by default `/opt/weaviate/data`) instead.